mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -1574,18 +1588,17 @@ sealed trait BinaryLogisticRegressionTrainingSummary extends BinaryLogisticRegre
  * @param objectiveHistory objective function (scaled loss + regularization) at each iteration.
  */
 private class LogisticRegressionTrainingSummaryImpl(
-    override val predictions: DataFrame,
-    override val probabilityCol: String,
-    override val predictionCol: String,
-    override val labelCol: String,
-    override val featuresCol: String,
-    val objectiveHistory: Array[Double])
+    predictions: DataFrame,
+    probabilityCol: String,
+    predictionCol: String,
+    labelCol: String,
+    featuresCol: String,
+    override val objectiveHistory: Array[Double])
```

Comment:
This `val` cannot be removed because in base class it is a `def` so override here is required.